Ginger/GingerCoreNET/ActionsLib/UI/VisualTesting/VRTAnalyzer.cs

```diff
@@ -320,7 +319,15 @@ private void TrackVRT()
                     switch (result.Status)
                     {
                         case TestRunStatus.New:
-                            mAct.Error += $"No baseline found, Please approve it on dashboard to create baseline.{System.Environment.NewLine}{result.Url}";
+                            if(mAct.CreateBaselineImage)
```

Comment:
Also make sure action is not failing on baseline upload

Reply:
Tested